pack_and_terraform_aws.bash

```diff
@@ -278,11 +278,11 @@ if echo "$runtf" | grep -iq "^y"; then
         google_client_id=$google_client_id \
         hmac_encryption_key=$hmac_encryption_key \
         gdcapi_secret_key=$gdcapi_secret_key \
-        db_password_userapi=$db_password_userapi \
+        db_password_fence=$db_password_fence \
         db_password_indexd=$db_password_indexd \
         db_password_gdcapi=$db_password_gdcapi \
         gdcapi_indexd_password=$gdcapi_indexd_password \
-        userapi_snapshot=$userapi_snapshot \
```

Comment:
See the comment on the aws_db_instance in kube.tf - do not remove userapi - just add fence, so we can deploy fence to existing commons without blowing up the service that are running in place, and we can easily roll back if problems come up.